samples/excel/85-preview-apis/tetrominos.yaml

```diff
@@ -0,0 +1,796 @@
+                    drawCoreMovementPane(session, tetrominoElement);
+
+                    containerElem.appendChild(tetrominoElement);
+                }
+
+                function drawCoreMovementPane(session, containerElem) {
+                    const sessionPaneElem = document.createElement("div");
+                    drawBoard(session, sessionPaneElem);
+
+                    containerElem.appendChild(sessionPaneElem);
+                }
+
+                function drawBoard(session, containerElem) {
+                    const boardPaneElem = document.createElement("div");
+                    boardPaneElem.classList.add("board");
+
+                    const rows = session.getRows();
+                    drawBlocks(rows, NUM_ROWS, NUM_COLS, boardPaneElem);
+
+                    containerElem.appendChild(boardPaneElem);
+                }
+
+                function drawConsolePane(session, isPaused, containerElem) {
+                    const consolePaneElem = document.createElement("div");
+                    consolePaneElem.classList.add("consolePane");
+
+                    const rowsClearedElem = drawRowsCleared(session, isPaused);
+                    const previewElem = drawSessionPreview(session);
+                    const usageElem = drawSessionUsage();
+
+                    consolePaneElem.appendChild(previewElem);
+                    consolePaneElem.appendChild(rowsClearedElem);
+                    consolePaneElem.appendChild(usageElem);
+
+                    containerElem.appendChild(consolePaneElem);
+                }
+
+                function drawRowsCleared(session, isPaused) {
+                    const rowsCleared = session.getRowsCleared();
+                    const rowsClearedElem = document.createElement("div");
+                    rowsClearedElem.classList.add("sessionRowsCleared");
+                    rowsClearedElem.innerHTML = "<p>Rows Cleared: " + rowsCleared + "</p>";
+                    if (isPaused) {
+                        rowsClearedElem.innerHTML += "<p>PAUSED</p>";
+                    }
+
+                    if (session.getSessionOver()) {
+                        rowsClearedElem.innerHTML += "<p>SESSION OVER</p>";
+                    }
+
+                    return rowsClearedElem;
+                }
+
+                function drawSessionPreview(session) {
+                    const piece = session.getNextPiece();
+                    const previewElem = document.createElement("div");
+                    previewElem.classList.add("preview");
+                    drawPreview(piece, 4, 4, previewElem);
+                    return previewElem;
+                }
+
+                function drawSessionUsage() {
+                    const usageElem = document.createElement("div");
+                    usageElem.classList.add("usage");
+                    usageElem.innerHTML =
+                        "<table>" +
+                        "<tr><th>Cursor Keys</th><td>Steer</td></tr>" +
```

Comment:
Some suggestions:

- Add a heading before this list of keys/commands that explains what the list is...e.g., maybe **How to play** or something along those lines...and consider adding some text after the heading, something along the lines of "Use your keyboard to play the game:"
- Line 1: Change **Cursor Keys** to **Left arrow key** and **Right arrow key** (assuming that's what you mean by "Cursor Keys") and split those into two separate lines -- one with description **Move shape left** and the other with description **Move shape right**
- Line 2: Split the **'a'/'d'** line into two separate lines -- one with title **'a' key** and the other with title **'d' key** -- and with descriptions of **Rotate shape clockwise** and **Rotate shape counterclockwise** as appropriate
- Line 3: Update title to **Space key** and description to **Let shape fall**
- Line 4: Update title to **Enter key**
- Line 5: Update title to **'R' key**